sdcm/utils/common.py

```diff
@@ -1483,3 +1483,10 @@ def download_encrypt_keys():
     for pem_file in ['CA.pem', 'SCYLLADB.pem', 'hytrust-kmip-cacert.pem', 'hytrust-kmip-scylla.pem']:
         if not os.path.exists('./data_dir/encrypt_conf/%s' % pem_file):
             ks.download_file(pem_file, './data_dir/encrypt_conf/%s' % pem_file)
+
+
+def normalize_ipv6_url(ip_address):
+    """adds square brackets on the IPv6 address in the URL"""
+    if ":" in ip_address:  # IPv6
```

Comment:
maybe we could use 
https://docs.python.org/3/library/ipaddress.html#module-ipaddress
```python
    if ipaddress.ip_address(ip_address).version == 6:  # IPv6
        return "[%s]" % ip_address
    return ip_address
```

Reply:
We can, but in future when we will completely move to  Python 3

Reply:
this needs backport to branch-3.2

Reply:
forgot the backports we need for this...